src/Robo/Common/EnvironmentDetector.php

```diff
@@ -348,11 +373,7 @@ public static function getAcsfDbName() {
    * @throws \ReflectionException
    */
   private static function getSubclassResults($functionName) {
-    if (!defined('DRUPAL_ROOT')) {
-      // Do nothing if called outside of a Drupal bootstrap.
-      return [];
-    }
-    $autoloader = require DRUPAL_ROOT . '/autoload.php';
+    $autoloader = require self::getRepoRoot() . '/vendor/autoload.php';
```

Comment:
`DRUPAL_ROOT` isn't the same as `self::getRepoRoot() . '/vendor'` but that's okay since Drupal's autoload.php is just a wrapper for the root autoloader.